api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,78 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	//
+	// +kubebuilder:default=Enabled
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	// +optional
+	Enabled string `json:"enabled,omitempty"`
+
+	// delayAfterAddSeconds sets how long after scale up the scale down evaluation resumes in seconds
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10000
+	// +kubebuilder:default=60
```

Comment:
Do we really want to default this at the server level? Would you ever expect to change this default? If you think in the future we may want to adjust this value without having to go and update every pre-existing object, then not having a default here is preferable and defaulting within the controller itself. We do this a lot in configuration APIs in openshift